src/components/Table/TableBody/RowActionsCell/_row-actions-cell.scss

```diff
@@ -11,6 +11,7 @@
   }
   .#{$iot-prefix}--row-actions-cell-btn--icononly {
     margin-left: 0;
+    color: black; // black by default
```

Comment:
Looks like it's Gray 100, not straight black

```suggestion
    color: $ui-05;
```

![image](https://user-images.githubusercontent.com/3360588/84954242-a4973500-b0ba-11ea-9e55-60bff9bed022.png)

Reply:
Actually with the `color` property it should probably be `$text-01`

Reply:
This style is no longer needed. Using `hasIconOnly` naturally gave the icon the correct color again